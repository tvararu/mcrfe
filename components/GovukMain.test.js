import renderer from "react-test-renderer";
import GovukMain from "./GovukMain";

describe("GovukMain", () => {
  it("renders correctly", () => {
    const component = renderer.create(<GovukMain>Lorem</GovukMain>);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
