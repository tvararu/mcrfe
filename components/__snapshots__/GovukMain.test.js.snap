// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GovukMain renders correctly 1`] = `
<main
  className="govuk-main-wrapper"
  id="main-content"
  role="main"
>
  Lorem
</main>
`;
